common/djangoapps/util/enterprise_helpers.py

```diff
@@ -0,0 +1,22 @@
+"""
+Utility library for working with the edx-enterprise app
+"""
+
+from django.conf import settings
+
+
+def get_enterprise_branding_info(provider_id):
+    """
+    Client API operation adapter/wrapper
+    """
+    if not enterprise_enabled():
+        return None
+    from enterprise import api as enterprise_api
+    return enterprise_api.get_enterprise_branding_info(provider_id=provider_id)
+
+
+def enterprise_enabled():
+    """
+    Returns boolean indication if enterprise app is enabled on not.
+    """
+    return settings.FEATURES.get('ENTERPRISE_APP', True)
```

Comment:
@bradenmacdonald It seems me, We should include this flag. I agree, When flag would be disabled then we should not show the Enterprise section in the admin panel. 
However, If you feel it is unnecessary at that point then we will need to catch the `ImportsError` whenever / where ever we are going to import Enterprise modules in `enterprise_helpers.py`. 
**OR**
We can do that instead of adding a feature flag. 

```
def enterprise_enabled():
    """
    Returns boolean
    """
    return 'enterprise' in settings.INSTALLED_APPS

```
Now let me know what you suggest ?